modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -146,15 +135,20 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}}{{^parent}}{{#ha
   {{#swagger1AnnotationLibrary}}
   @ApiModelProperty({{#example}}example = "{{{.}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}{{#isReadOnly}}readOnly = {{{isReadOnly}}}, {{/isReadOnly}}value = "{{{description}}}")
   {{/swagger1AnnotationLibrary}}
-  public {{>nullableDataType}} {{getter}}() {
+  public {{>dataType}} {{getter}}() {
```

Comment:
I think this will need a the bean validation added to it... which will require it to be inside the Optional/JsonNullable.

Maybe a second import of `dataTypeWithValidation` ?